python/tvm/auto_scheduler/relay_integration.py

```diff
@@ -48,7 +49,7 @@
 logger = logging.getLogger("auto_scheduler")
 
 
-def call_all_topi_funcs(mod, params, target, opt_level=3):
+def call_all_topi_funcs(mod, params, target, errors: List, opt_level=3):
```

Comment:
To be consistent let's have type annotation for all parameters in this file or nothing.

Reply:
Fair enough, I've removed type annotations inline but I've made it obvious what type errors is hopefully.